metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/native_sql_parser.py

```diff
@@ -41,7 +44,48 @@ def get_tables(native_query: str) -> List[str]:
         logger.debug(f"Type={type(tokens[from_index])}")
         if isinstance(tokens[from_index], sqlparse.sql.Identifier):
             # Split on as keyword and collect the table name from 0th position. strip any spaces
-            tables.append(tokens[from_index].value.split("as")[0].strip())
+            table: str = tokens[from_index].value.split("as")[0].strip()
+
+            if len(table.split(".")) == 1:
+                table = f"{default_schema}.{table}"
```

Comment:
this definitely won't create valid urns?

Reply:
It is not creating URN. It is only returning the table name in the form of <schema-name>.<table-name>. it is old code and existing test cases are working. The difference is; added `default_schema` as an argument.